doc/index.rst

```diff
@@ -66,6 +66,7 @@ Here is the citation in BibTeX format
     Quickstart<notebooks/Introduction>
     Example Gallery<examples/index>
     Cookbook<notebooks/InferenceDataCookbook>
+    Numba<notebooks/Numba>
```

Comment:
Completely unrelated. Should this toctree have the same elements as the sticky nav-bar on top of the website? 

If so this seems like a good ocasion to add `InferenceData<notebooks/XarrayforArviZ>` here and `("Numba", "notebooks/Numba"),` between lines 137-138 of `doc/conf.py`. Adding numba to the nav-bar is probably more important, otherwise it can be hard to find.